### DISCUSSION TITLE
Organization emojis

### DISCUSSION THREAD

**Discussion by suprstarrd** [community/community#16150] [Other Feature Feedback, Questions, & Ideas]
GitHub offers an easy way to use emojis in text-- shortcodes! Some examples of these are: 
* `:tada:` is :tada:
* `:+1:` is :+1:
* `:smile:` is :smile: 

These are only a few examples of [a very large list](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md).

GitHub *also* offers it's own custom emojis *exclusive to GitHub!*
* `:octocat:` is :octocat:
* `:bowtie:` is :bowtie:
* `:neckbeard:` is :neckbeard:
* `:shipit:` is :shipit:
* `:atom:` is :atom:
* `:electron:` is :electron:
* `:basecampy:` is :basecampy:
* `:trollface:` is :trollface:

Seems like GitHub can play DOOM, too! :feelsgood: :finnadie: :goberserk: :godmode: :hurtrealbad: :rage1: :rage2: :rage3: :rage4: :suspect:

I think it'd be wonderful for organizations to have the ability to upload their own custom emojis with shortcodes for use in their own repositories, adding a lot of fun and personality (or just branding) into what would end up being a mundane discussion otherwise.

**Comment by suprstarrd** [comment]
feeling cute, might bump and make a mockup later lol

**Comment by queenofcorgis** [comment]
Thanks for the suggestion @jbmagination 😄 :octocat:
